hawc/apps/assessment/views.py

```diff
@@ -91,122 +90,118 @@ def dispatch(self, request, *args, **kwargs):
-            )
+        visuals = (
+            apps.get_model("summary", "Visual").objects.count()
+            + apps.get_model("summary", "DataPivot").objects.count()
+        )
 
-            assessments_with_visuals = len(
+        assessments_with_visuals = len(
+            set(
+                models.Assessment.objects.order_by("-created")
+                .annotate(vc=Count("visuals"))
+                .filter(vc__gt=0)
+                .values_list("id", flat=True)
+            ).union(
                 set(
                     models.Assessment.objects.order_by("-created")
-                    .annotate(vc=Count("visuals"))
-                    .filter(vc__gt=0)
+                    .annotate(dp=Count("datapivot"))
+                    .filter(dp__gt=0)
                     .values_list("id", flat=True)
-                ).union(
-                    set(
-                        models.Assessment.objects.order_by("-created")
-                        .annotate(dp=Count("datapivot"))
-                        .filter(dp__gt=0)
-                        .values_list("id", flat=True)
-                    )
                 )
             )
+        )
 
-            counts = dict(
-                updated=updated,
-                users=users,
-                assessments=assessments,
-                references=references,
-                tags=tags,
-                references_tagged=references_tagged,
-                references_tagged_percent=percentage(references_tagged, references),
-                studies=studies,
-                assessments_with_studies=assessments_with_studies,
-                assessments_with_studies_percent=percentage(assessments_with_studies, assessments),
-                rob_scores=rob_scores,
-                studies_with_rob=studies_with_rob,
-                studies_with_rob_percent=percentage(studies_with_rob, studies),
-                endpoints=endpoints,
-                endpoints_with_data=endpoints_with_data,
-                endpoints_with_data_percent=percentage(endpoints_with_data, endpoints),
-                outcomes=outcomes,
-                results=results,
-                results_with_data=results_with_data,
-                results_with_data_percent=percentage(results_with_data, results),
-                iv_endpoints=iv_endpoints,
-                iv_endpoints_with_data=iv_endpoints_with_data,
-                iv_endpoints_with_data_percent=percentage(iv_endpoints_with_data, iv_endpoints),
-                visuals=visuals,
-                assessments_with_visuals=assessments_with_visuals,
-                assessments_with_visuals_percent=percentage(assessments_with_visuals, assessments),
-            )
-            cache_duration = 60 * 60 * 24  # one day
-            cache.set(key, counts, cache_duration)  # cache for one day
-            logger.info("Setting about-page cache")
-        return counts
+        counts = dict(
```

Comment:
I think this is currently always evaluated; we need to put this somewhere else so a cached version could be used. I'll make this edit real quick and merge this in.